stubs/stripe/stripe/api_resources/search_result_object.pyi

```diff
@@ -0,0 +1,33 @@
+from _typeshed import Self
+from collections.abc import Iterator
+from typing import Any, Generic, TypeVar
+from typing_extensions import Literal
+
+from stripe.stripe_object import StripeObject
+
+_T = TypeVar("_T")
+
+class SearchResultObject(StripeObject, Generic[_T]):
+    OBJECT_NAME: Literal["search_result"] = ...
+    url: str
+    has_more: bool
+    data: list[_T]
+    next_page: str
+    total_count: int
```

Comment:
Where are these attributes set?

Reply:
Sorry, these aren't documented in the library itself, but are in the API reference. These attributes should match the "Search response format" described [here](https://stripe.com/docs/api/pagination/search).

[This](https://github.com/stripe/stripe-python/blob/1ae42227/stripe/util.py#L169) is where a `SearchResultObject` would be instantiated, which eventually sets those values [here](https://github.com/stripe/stripe-python/blob/fa26fa57ff980eee12ab6b770df601e6dd7bef34/stripe/stripe_object.py#L222).

Reply:
Can't say I really follow the code here, but I'll take your word for it 😄